### DISCUSSION TITLE
Fullscreen image but responsive thumbnails?

### DISCUSSION THREAD

**Discussion by spetroll** [igordanchenko/yet-another-react-lightbox#26] [Q&A] [answered]
Hi,

I'am using yet-another-react-lightbox for my wedding gallery. Adding `srcset` to allow responsive images works perfectly. However, I would like to use responsive images only for the thumbnails, not the main-picture. 

When someone currently wants to save an image (which I guess many of my guests want to do), the image is downloaded mostly in the smaller size, especially when using it on mobile.

Is there an option I overlook?

**Comment by igordanchenko** [comment, marked as answer]
Hi,

Have you considered adding a dedicated "download" button? Here is one way you can implement the download feature - https://codesandbox.io/s/yet-another-react-lightbox-26-pnd59v?file=/src/App.js

But if you are dead set on the approach you described, here is how you can omit `srcSet` in the main picture - https://codesandbox.io/s/yet-another-react-lightbox-26-1-riigbz?file=/src/App.js

Hope this helps.

**Reply by spetroll** [reply]
Thanks Igor, the download button was the better solution. Thanks for suggesting and implementing it!
